load_tests/test_s3.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+import pandas as pd
+import pytest
+
+import awswrangler as wr
+
+
+@pytest.mark.repeat(1)
+@pytest.mark.parametrize("benchmark_time", [150])
+def test_s3_select(benchmark_time):
+    start = time.time()
+
+    path = "s3://ursa-labs-taxi-data/2018/1*.parquet"
+    wr.s3.select_query(
+        sql="SELECT * FROM s3object",
+        path=path,
+        input_serialization="Parquet",
+        input_serialization_params={},
+        scan_range_chunk_size=16 * 1024 * 1024,
+    )
+    end = time.time()
+
+    elapsed_time = end - start
+    assert elapsed_time < benchmark_time
+
+
+def test_s3_delete_objects(path, path2):
+    df = pd.DataFrame({"id": [1, 2, 3]})
+    objects_per_bucket = 505
+    paths = [f"s3://{path}delete-test{i}.json" for i in range(objects_per_bucket)] + [
+        f"s3://{path2}delete-test{i}.json" for i in range(objects_per_bucket)
+    ]
+    for path in paths:
+        wr.s3.to_json(df, path)
+    start = time.time()
+    wr.s3.delete_objects(path=paths)
+    end = time.time()
+    print(f"elapsed time: {end - start}")
```

Comment:
Did we make a decision on using `--durations=0` instead of repeating `time.time()` everywhere?

https://howchoo.com/python/how-to-measure-unit-test-execution-times-in-pytest

Reply:
You're right, looks like _path2list() was allowing me to be sloppy.

Reply:
This is an interesting case, because the file writes in the beginning of the test unfortunately take a lot longer than the deletes themselves. So we need some way to time just the delete.

Reply:
Another neat way of writing timers in Python I've used in the past is to extract the timer functionality into a context manager, which can then be used in a `with` block.

```python
from timeit import default_timer as timer

class ExecutionTimer():
    def __init__(self, msg):
        self.msg = msg
    def __enter__(self):
        self.before = timer()
    def __exit__(self, type, value, traceback):
        print(f"{self.msg}: {(timer() - self.before):.3f} sec")

with ExecutionTimer("elapsed time for wr.s3.delete_objects"):
    wr.s3.delete_objects(path=paths)
```

This block of code will then print the following:
```
elapsed time for wr.s3.delete_objects: 12.342 sec
```

Reply:
That's a nice idea, that way we could pull out the value out to do the assertion easily.

Reply:
I like it, let's give it a shot plz

Reply:
Implemented, thanks for the tip @LeonLuttenberger , will just need to confirm benchmarks are correct now.